SMDA: AttributeError: 'NoneType' object has no attribute 'items'

https://github.com/fireeye/capa/pull/568/checks?check_run_id=2686966250

```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests\fixtures.py:515: in do_test_feature_presence
    features = scope(extractor)
tests\fixtures.py:287: in inner
    f = get_function(extractor, va)
tests\fixtures.py:248: in get_function
    for f in extractor.get_functions():
capa\features\extractors\smda\__init__.py:29: in get_functions
    for function in self.smda_report.getFunctions():
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <smda.common.SmdaReport.SmdaReport object at 0x0000022FBAD11278>

    def getFunctions(self):
>       for _, smda_function in sorted(self.xcfg.items()):
E       AttributeError: 'NoneType' object has no attribute 'items'

c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\smda\common\SmdaReport.py:115: AttributeError
```

```
FAILED tests/test_smda_features.py::test_smda_features[a1982...-function=0x4014D0-characteristic(cross section flow)-True]
FAILED tests/test_smda_features.py::test_smda_features[al-khaser x64-function=0x14004B4F0-api(__vcrt_GetModuleHandle)-True]
```

--- a/setup.py
+++ b/setup.py
@@ -23,7 +23,7 @@ requirements = [
     "networkx==2.5.1",
     "ruamel.yaml==0.17.7",
     "vivisect==1.0.3",
-    "smda==1.5.17",
+    "smda==1.5.18",
     "pefile==2021.5.24",
 ]
 

--- a/tests/test_smda_features.py
+++ b/tests/test_smda_features.py
@@ -23,12 +23,6 @@ def test_smda_features(sample, scope, feature, expected):
     if scope.__name__ == "file" and isinstance(feature, capa.features.file.FunctionName) and expected is True:
         pytest.xfail("SMDA has no function ID")
 
-    if "a198216798ca38f280dc413f8c57f2c2" in sample and sys.platform == "win32":
-        pytest.xfail("SMDA bug tracked #585")
-
-    if "al-khaser_x64" in sample and sys.platform == "win32":
-        pytest.xfail("SMDA bug tracked #585")
-
     do_test_feature_presence(get_smda_extractor, sample, scope, feature, expected)
 
 

